openlibrary/plugins/upstream/utils.py

```diff
@@ -1,6 +1,6 @@
 import functools
 from dataclasses import dataclass
-from typing import Any
+from typing import TYPE_CHECKING, Any, Callable
 from collections.abc import Iterable, Iterator
```

Comment:
% `ruff --select=FLY,UP openlibrary/plugins/upstream/utils.py`
```
openlibrary/plugins/upstream/utils.py:3:1:
    UP035 [*] Import from `collections.abc` instead: `Callable`
```
```suggestion
from typing import TYPE_CHECKING, Any
from collections.abc import Callable, Iterable, Iterator
```

Reply:
Fixed!